### PR TITLE
Login/Create User using database

### PR DESCRIPTION
**Pull request by saundlj**
- Now able to create an account which stores user information in a database
- Users can login with email and password used when created account
- validation for unique email and username
- password confirmation
- Account page visible only for users  who are logged in 
- Navigation bar changes depending if user is logged in (shows logout vs login and register)
- flash messages are alerted at the top of the page template 
- updates requirements.txt with all packages

### CODE REVIEW

**Comment by saundlj** [comment]
If you guys are happy, i think this may be a good time to have our first milestone tag.
1st requirement of a user being able to create an account or login is completed on the backend but needs some work on the front end...

**Review by mjb44552 — COMMENTED** [review]
Looks very good. Only thing I might change is in the app/templates/CreateAccount.html document. From line 17 - 47 could the jinja template use a for loop to create the div objects (with class form-group)? Not sure if it's possible but if it is it would replace a lot of code that looks the same with like 4-8 lines.

**Comment by tomve1tch** [comment]
Seems to running well. Good job Luke. Happy to be pulled to origin as necessary.

File: app/templates/layout.html [hunk 1]
```diff
@@ -45,6 +45,17 @@
...
+            {% for category,message in message %}
+              <div class="alert alert-{{category}}">
+                {{message}}
+              </div>
+            {% endfor%}
+          {% endif %}
+        {% endwith %}
+
         {% block content %}{% endblock %}
     </div>
```
Comment (vittopan) [inline app/templates/layout.html:66]: Looking good

**Pull request merged**